selfdrive/car/subaru/carcontroller.py

```diff
@@ -1,19 +1,45 @@
+from common.numpy_fast import clip
 from opendbc.can.packer import CANPacker
 from selfdrive.car import apply_driver_steer_torque_limits
 from selfdrive.car.subaru import subarucan
 from selfdrive.car.subaru.values import DBC, GLOBAL_GEN2, PREGLOBAL_CARS, CarControllerParams
 
+ACCEL_HYST_GAP = 10  # don't change accel command for small oscilalitons within this value
+
+def accel_hysteresis(accel, accel_steady):
+
+  # for small accel oscillations within ACCEL_HYST_GAP, don't change the accel command
+  if accel > accel_steady + ACCEL_HYST_GAP:
+    accel_steady = accel - ACCEL_HYST_GAP
+  elif accel < accel_steady - ACCEL_HYST_GAP:
+    accel_steady = accel + ACCEL_HYST_GAP
+  accel = accel_steady
+
+  return accel, accel_steady
```

Comment:
is hysteresis needed?

Reply:
rapid small changes caused engine to be jerky, hysteresis improved things
openpilot long:
<img width="490" alt="Screenshot 2023-04-18 at 06 43 18" src="https://user-images.githubusercontent.com/148686/232665647-0c691e02-f5fb-4814-bd08-d3d070be016a.png">

stock:
<img width="488" alt="Screenshot 2023-04-18 at 06 37 36" src="https://user-images.githubusercontent.com/148686/232665680-3fe7f210-3d9c-47bb-b87c-d25ebd81c279.png">